src/cdk/testing/testbed/testbed-harness-environment.ts

```diff
@@ -67,13 +67,22 @@ function uninstallAutoChangeDetectionStatusHandler(fixture: ComponentFixture<unk
   }
 }
 
+/** Whether we are currently in the fake async zone. */
+function isInFakeAsyncZone() {
+  return Zone?.current.get('FakeAsyncTestZoneSpec') != null;
```

Comment:
If a `Zone` global variable isn't defined, this will still throw a `TypeError`. You'll have to guard it with `typeof Zone !== 'undefined'` to avoid the error, although I'm not sure how likely it is that we'll hit such a case.

Reply:
Changed it back to `!` which is what it actually was originally. I don't think we really support testing without zones currently anyways